[2025-09-17T18:45:01.698+0200] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-09-17T18:45:01.957+0200] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_api_to_bi.extract_customers scheduled__2025-09-17T16:30:00+00:00 [queued]>
[2025-09-17T18:45:01.982+0200] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_api_to_bi.extract_customers scheduled__2025-09-17T16:30:00+00:00 [queued]>
[2025-09-17T18:45:01.984+0200] {taskinstance.py:2306} INFO - Starting attempt 1 of 3
[2025-09-17T18:45:02.070+0200] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): extract_customers> on 2025-09-17 16:30:00+00:00
[2025-09-17T18:45:02.094+0200] {standard_task_runner.py:64} INFO - Started process 2208 to run task
[2025-09-17T18:45:02.100+0200] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'etl_api_to_bi', 'extract_customers', 'scheduled__2025-09-17T16:30:00+00:00', '--job-id', '120', '--raw', '--subdir', 'DAGS_FOLDER/etl_api_to_bi.py', '--cfg-path', '/tmp/tmpd3v6s9yp']
[2025-09-17T18:45:02.115+0200] {standard_task_runner.py:91} INFO - Job 120: Subtask extract_customers
[2025-09-17T18:45:02.435+0200] {task_command.py:426} INFO - Running <TaskInstance: etl_api_to_bi.extract_customers scheduled__2025-09-17T16:30:00+00:00 [running]> on host 1621268d5064
[2025-09-17T18:45:03.585+0200] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='data' AIRFLOW_CTX_DAG_ID='etl_api_to_bi' AIRFLOW_CTX_TASK_ID='extract_customers' AIRFLOW_CTX_EXECUTION_DATE='2025-09-17T16:30:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-09-17T16:30:00+00:00'
[2025-09-17T18:45:03.587+0200] {taskinstance.py:430} INFO - ::endgroup::
[2025-09-17T18:45:03.660+0200] {crypto.py:82} WARNING - empty cryptography key - values will not be stored encrypted.
[2025-09-17T18:45:03.661+0200] {base.py:84} INFO - Using connection ID 'postgres_default' for task execution.
[2025-09-17T18:45:03.727+0200] {python.py:237} INFO - Done. Returned value was: 1
[2025-09-17T18:45:03.732+0200] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-09-17T18:45:03.816+0200] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=etl_api_to_bi, task_id=extract_customers, run_id=scheduled__2025-09-17T16:30:00+00:00, execution_date=20250917T163000, start_date=20250917T164501, end_date=20250917T164503
[2025-09-17T18:45:03.915+0200] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2025-09-17T18:45:04.031+0200] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-09-17T18:45:04.050+0200] {local_task_job_runner.py:222} INFO - ::endgroup::
